src/gulpt-tasks/sass.js

```diff
@@ -3,19 +3,43 @@ const cleanCSS = require('gulp-clean-css');
 const sassProcessor = require('gulp-sass');
 const autoprefixer = require('gulp-autoprefixer');
 const gulpif = require('gulp-if');
+const {hashForContent} = require('../build/hash');
+const fs = require('fs');
 
 // We want to be using canonical Sass, rather than node-sass
 sassProcessor.compiler = require('sass');
 
 // Flags whether we compress the output, generate sourcemaps, etc
 const isProduction = process.env.ELEVENTY_ENV === 'prod';
 
+/**
+ *
+ * @param {Buffer} vinly.contents
+ * @param {string} vinly.basename
+ */
+const hashOutput = ({contents, basename}) => {
+  // gulp.dest() requires that we return a path to an output directory.
```

Comment:
Can you put this in the comment of the function (inside `/** */`?) I understand now but I imagined the internal comment to be a comment on _how_ this function worked, not what it did.